terraform/constellation-cluster/variables.tf

```
@@ -84,10 +107,10 @@ variable "gcp_config" {
 
 variable "image" {
   type        = string
-  description = "The node image reference or semantical release version."
+  description = "The node image reference or semantic release version."
```

This allows to leave the field empty too and follows the same defaults we have in the CLI config.
```suggestion
  description = "The node image reference or semantic release version."
  default     = "@@CONSTELLATION_VERSION@@"
```

---

Any argument in favor of this except one less input? I think we should have an explicit image input to be aligned with the config. But no strong opinion.

---

One less input with sensible default. This is what `config generate` provides too.  It can still be configured. How does it not align with the config?

---

You're right. I thought we didn't fill it in the config.